redbot/core/commands/commands.py

```diff
@@ -398,6 +398,44 @@ def clear_rule_for(
         return old_rule, new_rule
 
 
+Command.error.__doc__ = """
+A decorator that registers a coroutine as a local error handler.
+
+A local error handler is an :func:`.on_command_error` event limited to
+a single command.
+
+The on_command_error event is still dispatched 
+for commands with a dedicated error handler.
+
+Red's global error handler will ignore commands with a registered error handler.
+
+To have red handle specific errors with the default behavior, 
+call ctx.bot.on_command_error with `unhandled_by_cog` set to True
+
+For example:
+
+    .. code-block:: python
+
+        @a_command.error
+        async def a_command_error_handler(self, ctx, error):
+
+            if isisntance(error, MyErrrorType):
+                self.log_exception(error)
+            else:
+                await ctx.bot.on_command_error(ctx, error, unhandled_by_cog=True)
+
+Parameters
+-----------
+coro : :ref:`coroutine <coroutine>`
+    The coroutine to register as the local error handler.
+
+Raises
+-------
+discord.ClientException
+    The coroutine is not actually a coroutine.
+"""
```

Comment:
Could you please create the `error(self, coro)` method on our command class to just call `super().error(coro)`, and then move this docstring there? That way it'll show up in our docs, and is more obvious in the code.

Reply:
I've created the method and moved the docstring.